Makefile

```diff
@@ -24,21 +25,18 @@ check:
 	find . -name "*.pyc" -exec rm {} \;
 	PYTHONPATH=$(CURDIR) PYTHONDONTWRITEBYTECODE=1 python3 -m pytest --verbose --showlocals $(TESTS_TARGET)
 
+requre_data_cleanup:
+	requre-patch purge --replaces "copr.v3.helpers:login:str:somelogin" --replaces "copr.v3.helpers:token:str:sometoken" $(TESTS_RECORDING_PATH)/test_data/*/*yaml
+	requre-patch create-symlinks $(CURDIR)/$(TESTS_RECORDING_PATH)/test_data/
+
 # example: TESTS_TARGET=tests/unit/test_api.py make check_in_container
 check_in_container: tests_image
-	$(CONTAINER_ENGINE) run --rm -ti \
```

Comment:
@bcrocker15 is working on streamlining the check* targets so please make sure your changes are aligned: https://github.com/packit/packit/pull/1230

Reply:
Seems that these two are more less independent and orthogonal.

Have to talk to @bcrocker15 if it is fine.